apps/ios/Shared/Views/Chat/ChatView.swift

```diff
@@ -130,18 +131,21 @@ struct ChatView: View {
                     }
                 } else if case let .group(groupInfo) = cInfo {
                     Button {
-                        Task {
-                            let groupMembers = await apiListMembers(groupInfo.groupId)
-                            await MainActor.run {
-                                ChatModel.shared.groupMembers = groupMembers
-                                showChatInfoSheet = true
-                            }
-                        }
+                        Task { await loadGroupMembers(groupInfo) { showChatInfoSheet = true } }
                     } label: {
                         ChatInfoToolbar(chat: chat)
                     }
                     .appSheet(isPresented: $showChatInfoSheet) {
-                        GroupChatInfoView(chat: chat, groupInfo: groupInfo)
+                        GroupChatInfoView(
+                            chat: chat,
+                            groupInfo: Binding(
```

Comment:
This is needed to make sure group info updates on received events when view is opened, and also that updates from inside the view are reflected outside.